pkgs/development/compilers/nim/default.nix

```diff
@@ -105,7 +105,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Statically typed, imperative programming language";
-    homepage = https://nim-lang.org/;
+    homepage = "https://nim-lang.org/";
```

Comment:
Any particular reason for turning this into a string?

Reply:
@peterhoeg https://github.com/NixOS/rfcs/pull/45

Reply:
I wasn't aware of that - thanks!